PortalSwift/Sources/PortalSwift/Utils/Blockchains/PortalBlockchain.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  PortalBlockchain.swift
+//
+//
+//  Created by Blake Williams on 3/30/24.
+//
+
+import Foundation
+
+public class PortalBlockchain {
+  static let mainnetReferences: [PortalNamespace: String] = [
+    .eip155: "1",
+  ]
+  static let namespaceCurves: [PortalNamespace: PortalCurve] = [
+    .eip155: .SECP256K1,
+    .solana: .ED25519,
+  ]
+  static let namespaceSignerMethods: [PortalNamespace: [PortalRequestMethod]] = [
+    .eip155: [
+      .eth_sendTransaction,
+      .eth_sign,
+      .eth_signTransaction,
+      .eth_signTypedData_v3,
+      .eth_signTypedData_v4,
+      .personal_sign,
```

Comment:
We should add an eth_ in front of any eth methods that dont already have it. 
I think we would just need to add 
`eth_personal_sign = "personal_sign"` otherwise it will be confusing for other chains

Reply:
I'll likely do that as part of the Solana updates, since currently this only supports Ethereum (plus we're using this enum as an argument, so it'll be a pretty swap fix in the future).

Reply:
yea that is why i mentioned it cause i made that change as apart of the solana work too. Sounds good